website/docs/terms/cte.md

```diff
@@ -1,30 +1,186 @@
 ---
 id: cte
-title: CTE
+title: CTE in SQL
 displayText: CTE  
-hoverSnippet: CTE is one of those SQL specific acronyms that is impossible to Google.
+hoverSnippet: A Common Table Expression (CTE) is a temporary result set that can be used in a SQL query. Creating a CTE is essentially like making a temporary view that you can access throughout the rest of the query you are writing.
 ---
 
-CTE is one of those SQL specific acronyms that is impossible to Google. It stands for common <Term id="table" /> expression, but the name itself doesn’t feel super self explanatory.. A CTE is used in order to enable readability within code. It is used to group little query snippets, and reference them all within one large piece of work. Each CTE is a common-table-expression because you are creating and naming a snippet that you will use more than once within your code so that you do not need to repeat, but instead just reference the common piece of code. 
+In a formal sense, a Common Table Expression (CTE), is a temporary result set that can be used in a SQL query. In a less formal, more human-sense, we like to think of a CTE as a separate, smaller query within the larger query you’re building up. Creating a CTE is essentially like making a temporary <Term id="view" /> that you can access throughout the rest of the query you are writing. Ultimately, you can use CTEs to break up complex queries into simpler blocks of code that can connect and build on each other.
 
-Common Table Expression - Creating an ephemeral table to reference downstream in your SQL query for ease of understanding / improved query readability. In my experience, these are extra helpful when you need to operate over aggregates.
+There are two-types of CTEs: recursive and non-recursive. This glossary focuses on non-recursive CTEs.
 
-CTEs are really helpful for troubleshooting. You can easily track each step of your process to see where something went wrong.
+## Why you should care about CTEs
 
+Have you ever read through a query and thought:
+
+- “What does this part of the query do?” 
+- “What are all the sources referenced in this query? Why did I reference this dependency?”
+- “My query is not producing the results I expect and I’m not sure which part of the query is causing that.”
+
+These thoughts often arise when we’ve written SQL queries and models that utilize complex business logic, references and joins multiple upstream dependencies, and are not outputting expected results. In a nutshell, these thoughts can occur often when you’re trying to write data models!
```

Comment:
Yeah that's true. I don't think Kira was trying to make a list here, I think its just a wordy sentence. I played around with the wording a bit. Does that read better to you?

Reply:
Yes! Thank you! Riffing on your version, we could rewrite to the 2nd person to match the last sentence of that paragraph:
> You might think these thoughts when writing SQL that utilizes complex business logic and references and joins multiple upstream dependencies, but the queries are not outputting expected results. In a nutshell, these thoughts can occur often when you’re trying to write data models!

I'm also curious is the "they" referring to "queries?" (I assumed yes when I rewrote it above). When we translate the docs, it will be super useful to be specific with the noun because it will help reduce ambiguity.